delivery_roulier/tests/test_delivery_roulier.py

```diff
@@ -55,6 +55,7 @@ def setUp(self):
                 "delivery_type": "test",
                 "product_id": delivery_product.id,
                 "carrier_account_id": self.account.id,
+                "automatic_package_creation_at_delivery": True,
```

Comment:
@florian-dacosta I've updated the test here to get the same behavior as before.

Reply:
A migration script is missing to set the value also on existing,
otherwise it is a breaking change.